integration/testnet/network.go

```diff
@@ -583,19 +584,43 @@ func BootstrapNetwork(networkConf NetworkConfig, bootstrapDir string) (*flow.Blo
 		return nil, nil, nil, nil, fmt.Errorf("failed to setup keys: %w", err)
 	}
 
+	// write private key files for each node
```

Comment:
moved up to separate out staked vs non-staked nodes before calling `runDKG` for only the staked nodes.